src/main/java/com/mitchej123/hodgepodge/config/FixesConfig.java

```diff
@@ -409,6 +409,16 @@ public class FixesConfig {
     @Config.RequiresMcRestart
     public static boolean fixExtraUtilitiesUnEnchanting;
 
+    @Config.Comment("Remove rain from the Last Millenium (Extra Utilities)")
```

Comment:
These aren't fixes, but GTNH gameplay tweaks to the last millenium. The original dimension is a reference to Chrono Trigger and Hodgepodge should default to `false` for them IMO.

Reply:
In a dimension with no sun, moon or clouds, rain just seems like an bug (or oversight) and people would be more surprised of the presence of rain rather than its absence.

I don't think it matters either way anyways.